srcs/app/server/FDsets.cpp

```diff
@@ -0,0 +1,41 @@
+#include <FDsets.hpp>
+#include <cstring>
+
+FDsets::FDsets() {
+	FD_ZERO(&read_set_);
+	FD_ZERO(&write_set_);
+}
+
+void	FDsets::removeFromReadSet(int fd) {
+	FD_CLR(fd, &read_set_);
+}
+
+void	FDsets::removeFromWriteSet(int fd) {
+	FD_CLR(fd, &write_set_);
+}
+
+void	FDsets::addToWriteSet(int fd) {
+	FD_SET(fd, &write_set_);
+}
+
+void	FDsets::addToReadSet(int fd) {
+	FD_SET(fd, &read_set_);
+}
+
+fd_set	*FDsets::getReadSet() {
+	std::memcpy(&tmp_read_set_, &read_set_, sizeof(read_set_));
+	return &tmp_read_set_;
+}
+
+fd_set	*FDsets::getWriteSet() {
+	std::memcpy(&tmp_write_set_, &write_set_, sizeof(read_set_));
```

Comment:
probably the sizeof is going to be the same for read and write sets, but maybe it would be more readable to have here sizeof(write_set_)